run.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+from feito.github import API
+from feito import Prospector
+from feito import Messages
+from feito import Repository
+
+GITHUB_PR_ID = os.getenv('GITHUB_PR_ID')
+REPO_USERNAME = os.getenv('REPO_USERNAME')
+TOKEN = os.getenv('TOKEN')
+
+repo = Repository()
+
+# TODO: Transform this into a cli
+def run():
+    analysis = Prospector(repo).run()
+    messages = Messages(analysis).commit_format()
+    if messages == []:
```

Comment:
No need to check this if we're using `for message in messages`.

Reply:
Yes, of course

![source](https://user-images.githubusercontent.com/21130697/36632788-f59539d8-1969-11e8-848b-78b1bce23055.gif)